### PR TITLE
Integrate vscode-r-lsp

### LINKED ISSUE #691: Integrate vscode-r-lsp into vscode-R
Since the project has been transferred to REditorSupport as suggested in #98, it is straightforward to integrate vscode-r-lsp into vscode-R, just like I did in https://github.com/REditorSupport/vscode-R/issues/98#issuecomment-638852903.

The vscode-r-lsp has some major changes since then, therefore I need to re-work on it if we still think it is a good idea.

Also, we need to discuss what should be done after the integration on our side and user side, e.g. announcement, deprecation, etc.

### PR DESCRIPTION
**Pull request by renkun-ken**
Closes #691 

1. Disable vscode-r-lsp
2. Open an R project and everything should behave in the same way with using vscode-r-lsp.

### CODE REVIEW

**Comment by ElianHugh** [comment]
Tested on WSL (Ubuntu) and everything seems to work nicely as before. No errors in dev console or output channel either

**Comment by renkun-ken** [comment]
@randy3k @Ikuyadeu Do you think we could merge this PR before the next release to the marketplace?

**Comment by Ikuyadeu** [comment]
We haven't released it since I moved from Ikuyadeu.
I think that there is no problem once we check the display of the Marketplace.

**Comment by randy3k** [comment]
How are we going to deprecate `vscode-r-lsp`?

**Comment by renkun-ken** [comment]
> How are we going to deprecate `vscode-r-lsp`?

Now before vscode-R starts the language server, it checks if vscode-r-lsp is enabled. If so, vscode-R does not start the language server and advice the user to uninstall vscode-r-lsp.

Then I'll add a visible note to the README of vscode-r-lsp saying that the extension has been merged into vscode-R and will be unpublished at some point. Some time later, we could officially unpublish vscode-r-lsp and no user could search or install it from within vscode extensions panel.

**Review by randy3k — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,42 +3,42 @@
...
 
-* [languageserver](https://github.com/REditorSupport/languageserver): An R package that implements the Language Server Protocol for R to provide a wide range of language analysis features such as auto-completion, function signature, documentation, symbol highlight, document outline, code formatting, symbol hover, diagnostics, go to definition, find references, etc.
-
-* [vscode-r-lsp](https://marketplace.visualstudio.com/items?itemName=REditorSupport.r-lsp): A VS Code extension of R LSP Client to communicate between VS Code and R Language Server.
-
 * [VSCode-R-Debugger](https://github.com/ManuelHentschel/VSCode-R-Debugger): A VS Code extension to support R debugging capabilities.
 
 * [httpgd](https://github.com/nx10/httpgd): An R package to provide a graphics device that asynchronously serves SVG graphics via HTTP and WebSockets.
 
+Go to the installation wiki pages ([Windows](https://github.com/Ikuyadeu/vscode-R/wiki/Installation:-Windows) | [macOS](https://github.com/Ikuyadeu/vscode-R/wiki/Installation:-macOS) | [Linux](https://github.com/Ikuyadeu/vscode-R/wiki/Installation:-Linux)) for more detailed instructions.
```
Comment (eitsupi) [inline README.md:32]: Old URL?

Comment (renkun-ken) [reply README.md]: A bit behind the master. Resolved.

**Pull request merged**